toplevel/opttopmain.ml

```diff
@@ -247,6 +247,14 @@ module Options = Main_args.Make_opttop_options (struct
   let anonymous = file_argument
 end);;
 
+let () =
+  let extra_paths =
+    match Sys.getenv "OCAMLTOP_INCLUDE_PATH" with
+    | exception Not_found -> []
+    | s -> Misc.split_path_contents s
+  in
+  Clflags.include_dirs := List.rev_append extra_paths !Clflags.include_dirs
```

Comment:
If `!Clflags.include_dirs` holds the dirs as found on the `cli` which according to the manual are used from left to right then I think the right invocation here should be `List.append !Clflags.include_dirs extra_path` so that `OCAMLTOP_INCLUDE_PATH` is only used as a last resort.

Reply:
The variable `Clflags.include_dirs` stores the directories in inverse order (each successive `-I` adds its argument to the front of the list) and gets reversed before being used (see https://github.com/ocaml/ocaml/blob/trunk/driver/compmisc.ml#L38), so I *think* this is OK.

Reply:
Yes agreed.